pymchelper/utils/pld2sobp.py

```diff
@@ -12,6 +12,26 @@
 logger = logging.getLogger(__name__)
 
 
+def dedx_air(energy):
+    """
+    Calculate the mass stopping power of protons in air following ICRU 49.
+    Valid from 1 to 500 MeV only.
+
+    :params energy: Proton energy in MeV
+    :returns: mass stopping power in MeV cm2/g
+    """
+    from numpy import log
```

Comment:
Do we really need numpy here ?
Standard library has all what we need https://docs.python.org/2/library/math.html#power-and-logarithmic-functions

If you really want to stick to numpy, then these imports should go to module level.
It doesn't make sense to call import in a loop (`dedx_air` is called many times)
I'd also recommend to do `import numpy as np` and then use `np.exp` and `np.log` in order not to shadow methods from standard python `math` library.